pkg/couchdb/database/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@librocco/database",
+	"version": "0.0.1",
+	"main": "./dist/index.js",
+	"exports": {
+	".": {
+		"import": "./dist/index.es.js",
+		"require": "./dist/index.js",
+		"default": "./dist.index.es.js"
+	}
+	},
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
```

Comment:
This can also be removed `vite preview` (it's used for built apps, e.g. svelte apps)